src/components/Profile.vue

```diff
@@ -1,10 +1,8 @@
 <template id="profile">
 
     <div class="profile-bar">
-        <div>Managing <router-link v-bind:to="'/user'"><strong>{{ selectedTrader.name }}</strong></router-link></div>
-        <transition name="fade" mode="out-in">
-            <div :key="voucherCount" v-if="vouchersAdded"><router-link v-bind:to="'/payment'"><strong>{{ voucherCount }}</strong> voucher<span v-if="voucherCount > 1">s</span> waiting</router-link></div>
-        </transition>
+        <div><router-link v-bind:to="'/account'"><strong>{{ selectedTrader.name }}</strong></router-link> <router-link v-bind:to="'/user'">(Change trader)</router-link></div>
```

Comment:
- (change trader) is good, but it's a function that the trader name used to have. I see why you've moved it to make it an explicit option, it was otherwise not obvious how to swap traders.

- The trader name still doesn't advertise it's active or where it goes; hidden discoverable? given the My Account page alreday exists and is advertised, why link the name in this way?